gpt_engineer/file_selector.py

```diff
@@ -196,10 +198,12 @@ def ask_for_selection(self) -> List[str]:
                         start = int(start_str)
                         end = int(end_str)
                         for num in range(start, end + 1):
-                            selected_paths.append(str(self.selectable_file_paths[num]))
+                            selected_paths.append(
+                                str(self.selectable_file_paths[num]))
```

Comment:
I think this makes the CI break.

Have you configured your pre-commit hooks like described [here](https://github.com/AntonOsika/gpt-engineer/blob/main/.github/CONTRIBUTING.md)?

I think just

`pre-commit install`